pkg/gds/certman/certs.go

```diff
@@ -824,6 +835,10 @@ vaspsLoop:
 	}
 }
 
+func RetrieveCertID(identityCert *pb.Certificate) string {
+	return fmt.Sprintf("%X", identityCert.SerialNumber)
+}
```

Comment:
I do like this helper, however I think we can make it a bit more useful by moving it to the `models` package (`pkg/models/v1`) so other packages can use it. Then, let's replace all instances where we are encoding the serial number with this helper method so that we can standardize that a bit more (basically just search the repo for `fmt.Sprintf("%X`).

Also since we're moving it to a different package can we change the name of the method to `GetCertID` or something along those lines to better match the naming conventions there?

Reply:
That's a good idea!